mqt/qmap/bindings.cpp

```diff
@@ -475,12 +475,18 @@ PYBIND11_MODULE(pyqmap, m) {
 
   auto tableau = py::class_<cs::Tableau>(
       m, "Tableau", "A class for representing stabilizer tableaus.");
-  tableau.def(py::init<std::size_t>(), "n"_a,
+  tableau.def(py::init<std::size_t, bool>(), "n"_a,
+              "use_fullsize_tableau"_a = false,
```

Comment:
This still has the old name that needs to be updated.
```suggestion
              "include_destabilizers"_a = false,
```